gradle/spec/dependabot/gradle/file_fetcher/settings_file_parser_spec.rb

```diff
@@ -24,19 +24,28 @@
       it "includes the additional declarations" do
         expect(subproject_paths).to match_array(%w(app))
       end
+    end
 
-      context "when kotlin" do
-        let(:settings_file) do
-          Dependabot::DependencyFile.new(
-            name: "settings.gradle.kts",
-            content: fixture("settings_files", fixture_name)
-          )
-        end
-        let(:fixture_name) { "settings.gradle.kts" }
+    context "with various call styles" do
+      let(:fixture_name) { "call_style_settings.gradle" }
 
-        it "includes the additional declarations" do
-          expect(subproject_paths).to match_array(%w(app))
-        end
+      it "includes the additional declarations" do
+        expect(subproject_paths).to match_array(%w(function_without_space function_with_space
+                                                         implicit implicit_with_many_spaces))
```

Comment:
looks like there's a rubocop linting failure here
```suggestion
        expect(subproject_paths).to match_array(
          %w(function_without_space function_with_space implicit implicit_with_many_spaces)
        )
```

Reply:
That format makes more sense, yeah. It's a while since I've touched ruby :)